### DISCUSSION TITLE
Setting up ArcGIS Assistant as an app in Enterprise

### DISCUSSION THREAD

**Discussion by webturtles** [EsriPS/arcgis-assistant-feedback#85] [Q&A]
Hi
I feel like I am missing a setting when trying to set up the assistant to access an enterprise portal, so just wondering what the app settings should look like in terms of incoming URL and redirect URI.  For context I use the old assistant in local portals no problem (using PKI/IWA option) and both old/new fine in AGOL.

I have added https://assistant.esri-ps.com under security for CORS, but get similar to others when it tries to handshake with AD credentials.
![image](https://github.com/EsriPS/arcgis-assistant-feedback/assets/29141879/7b0becbc-9024-4fc6-aec9-b995077f93e8)
Looking at the browser console, the main two errors look like:
![image](https://github.com/EsriPS/arcgis-assistant-feedback/assets/29141879/9bfcf5f4-f7e4-4a54-8a70-116f71c9cf80)

Any pointers in other things to check please?
Thanks

**Comment by tshewring-he** [comment]
Hi @webturtles  I am having the same issue. Did you ever get this resolved?

**Reply by webturtles** [reply]
Hi, no I didn't, sorry (works great in AGOL though!).  The old one works for most of our immediate needs, so had parked it until running 11.4, then will try again (as we will be replacing wab apps with experience ones).

**Reply by tshewring-he** [reply]
Thanks @webturtles I've just got it working but I have to use the 7443 address for Enterprise Portal (bypass WebAdaptor) when signing in via the ArcGIS Assistant. This doesn't require an entry in Security>Allow origins

**Reply by webturtles** [reply]
Useful to know, thanks.
